src/python-common/ceph/tests/test_drive_group.py

```diff
@@ -162,14 +169,15 @@ def test_ceph_volume_command_6():
     spec = DriveGroupSpec(placement=PlacementSpec(host_pattern='*'),
                           data_devices=DeviceSelection(rotational=False),
                           journal_devices=DeviceSelection(rotational=True),
-                          journal_size='500M',
+                          journal_size=SizeMatcher.str_to_byte('500M'),
                           objectstore='filestore'
                           )
+    spec.validate()
     inventory = _mk_inventory(_mk_device(rotational=True)*2 + _mk_device(rotational=False)*2)
     sel = drive_selection.DriveSelection(spec, inventory)
     cmd = translate.to_ceph_volume(spec, sel, []).run()
     assert cmd == ('lvm batch --no-auto /dev/sdc /dev/sdd '
-                   '--journal-size 500M --journal-devices /dev/sda /dev/sdb '
+                   '--journal-size 500000000.0 --journal-devices /dev/sda /dev/sdb '
```

Comment:
this looks odd. is this intended? @jschmid1

Reply:
I think with #34740 passing sizes in `M/G/T` format is valid again.

For now, we should be on the safe side with passing it as `float`